tox.ini

```diff
@@ -1,32 +1,32 @@
 [tox]
 envlist =
-    py27,py33,py34,py35,py36,pypy,pypy3,coverage,docs
+    py27,py34,py35,py36,pypy,pypy3
 
 [testenv]
+usedevelop = true
 commands =
     coverage run -m zope.testrunner --test-path=src []
+    coverage run -a -m sphinx -b doctest -d {envdir}/.cache/doctrees docs {envdir}/.cache/doctest
 deps =
-    .[test]
+    .[test,docs]
     coverage
 setenv =
    COVERAGE_FILE=.coverage.{envname}
 
 [testenv:coverage]
+usedevelop = true
 setenv =
   COVERAGE_FILE=.coverage
-skip_install = true
 commands =
     coverage erase
     coverage combine
-    coverage report
-    coverage html
-    coverage xml
+    coverage report -i
+    coverage html -i
+    coverage xml -i
 
 [testenv:docs]
 basepython =
     python2.7
 commands =
     sphinx-build -b html -d docs/_build/doctrees docs docs/_build/html
     sphinx-build -b doctest -d docs/_build/doctrees docs docs/_build/doctest
-deps =
-    Sphinx
```

Comment:
The docs extra is installed by default in the base `[testenv]`  because it runs the doctests, so this was no longer needed.